requirements.txt

```
@@ -22,6 +22,10 @@ importlib-metadata>=6.0.1,<=6.0.1  # Pinned to fix opentelemetry dependency solv
 jinja2>=3.1,<=3.1.6
 json_stream>=2.3.2,<2.4
 jq>=1.6.0,<1.7.0
+# pycares is only a transitive dependency, but there is a combination of versions that expresses a bug:
+# pycares==4.9 + aiodns==3.2.0 + python<3.12
+pycares<4.9;python_version>='3.9'
+pycares<4.5;python_version<'3.9'
```

Shouldn't this still be?
```suggestion
pycares<4.9;python_version<'3.12'
pycares<4.5;python_version<'3.9'
```